### PR TITLE
Refining the README introduction

### PR DESCRIPTION
**Pull request by chadly**
Working on the pitch in the README. The idea is to sell the idea of writing jobs the "Runly way" and then to upsell "the platform" after that. [Read it as it will appear on GitHub](https://github.com/runlyio/core-dotnet/tree/intro#readme).

Once #7 is merged, I want to add an animated SVG of the CLI in action similar to the [Node.js repo](https://github.com/runlyio/core-nodejs).

To anticipate feedback I think I know @WillSoss will have, the _The Runly Platform sounds cool. Why isn't it OSS too?_ section is meant to be more playful and make us seem more "human." People like to do business with people, so if we can pierce the "corporate business selling software" veil and appeal to a dev's sense of empathy, I think we will be more successful.

### CODE REVIEW

**Comment by chadly** [comment]
Once this is in, the goal would be to update the landing pages (runlyio/web#68) with the same or similar messaging and then start to promote this shit.

**Comment by chadly** [comment]
@WillSoss I tweaked the OSS section. It now avoids the initial question while still providing a good segue to the "support us" stuff.

**Comment by chadly** [comment]
Terminal animation is ready now

File: README.md [hunk 1]
```diff
@@ -58,6 +74,48 @@ Using command line flags or JSON config, this can then easily be multi-threaded
 
 See [more in-depth examples](./examples) or [start reading the docs](https://www.runly.io/docs/).
 
+## Why Write Code Like This?
+
+A lot of problems that us software developers are tasked with solving everyday boil down to processing lists of items. Some examples could be:
```
Comment (WillSoss) [inline README.md:79]:
```suggestion
A lot of problems that software developers are tasked with solving everyday boil down to processing lists of items. Some examples could be:
```

File: README.md [hunk 2]
```diff
@@ -4,9 +4,21 @@
 
 Runly is an opinionated framework for writing multi-threaded jobs. You focus on writing your business logic and Runly gives you a CLI, multi-threading, retries, and more out-of-the-box.
```
Comment (WillSoss) [inline README.md:5]:
```suggestion
Runly is a toolset for simplifying complex applications by writing compartmentalized, testable code. You focus on writing your business logic and Runly gives you a CLI, multi-threading, retries, and more out-of-the-box.
```

File: README.md [hunk 3]
```diff
@@ -58,6 +74,48 @@ Using command line flags or JSON config, this can then easily be multi-threaded
 
 See [more in-depth examples](./examples) or [start reading the docs](https://www.runly.io/docs/).
 
+## Why Write Code Like This?
+
+A lot of problems that us software developers are tasked with solving everyday boil down to processing lists of items. Some examples could be:
+
+* importing and processing records from large uploaded CSV files into a database
```
Comment (WillSoss) [inline README.md:81]:
```suggestion
* importing and processing records from large files into a database
```

File: README.md [hunk 4]
```diff
@@ -58,6 +74,48 @@ Using command line flags or JSON config, this can then easily be multi-threaded
...
+* downloading or uploading import/export files automatically on a schedule
```
Comment (WillSoss) [inline README.md:82]:
```suggestion
* downloading or uploading import/export files on a schedule automatically
```

File: README.md [hunk 5]
```diff
@@ -58,6 +74,48 @@ Using command line flags or JSON config, this can then easily be multi-threaded
...
+* generating PDFs from a list of records in a database
+* handling ETL scenarios
+
+A Runly job takes this pattern and standardizes it into a base `Job` class. Building and running jobs in a predictable way like this allows us to build lots of goodies on top of your core jobs. Things like multi-threading, scaling work across different machines, and resiliency and fault-tolerance become almost trivial for you to implement.
+
+In fact, we have done exactly that and built a whole platform on top of this core OSS project.
+
+## The Runly Platform
+
+You can turbocharge your job apps using the [Runly Platform](https://www.runly.io/platform/). The Runly Platform is a SaaS which helps shepherd your jobs from deployment to monitoring and scaling in production.
```
Comment (WillSoss) [inline README.md:93]:
```suggestion
You can turbocharge your job apps using the [Runly Platform](https://www.runly.io/platform/). The Runly Platform is SaaS which helps shepherd your jobs from deployment to monitoring and scaling in production.
```

File: README.md [hunk 6]
```diff
@@ -58,6 +74,48 @@ Using command line flags or JSON config, this can then easily be multi-threaded
...
+
+* Automatically deploy your jobs to your machines or the cloud as NuGet packages using tools you already know and love.
```
Comment (WillSoss) [inline README.md:95]:
```suggestion
* Automatically deploy your jobs to your existing machines or the cloud as NuGet packages using tools you already know and love.
```

File: README.md [hunk 7]
```diff
@@ -58,6 +74,48 @@ Using command line flags or JSON config, this can then easily be multi-threaded
...
+* Use familiar REST concepts to queue and schedule background jobs from your web or mobile app.
+* Easily integrate progress bars and job status into your app using [Runly UI](https://www.runly.io/ux/).
+* Use CRON expressions for fine-grained control of job scheduling.
+* Get insights into job failures and performance issues before your users notice.
+
+You can get started on a [generous free tier](https://www.runly.io/pricing/) with no commitment and no credit card. [Get started for free using your GitHub account](https://www.runly.io/dashboard/) or learn more about [how it works](https://www.runly.io/docs/).
+
+## Is this project sustainable?
+
+[Sustainability in OSS is a problem](https://twitter.com/UdiDahan/status/1282925432166285312). We are trying to work on this project full-time and in order to do that, we have to feed our families. We think providing value via a paid service is the best way to do that while still solving problems for people with the core OSS offering.
```
Comment (WillSoss) [inline README.md:105]:
```suggestion
[Sustainability in OSS is a problem](https://twitter.com/UdiDahan/status/1282925432166285312). We are working on this project full-time and in order to keep doing that, we have to feed our families. We think providing value via a paid service is the best way to do that while still solving problems for people with the core OSS offering.
```

File: README.md [hunk 8]
```diff
@@ -58,6 +74,48 @@ Using command line flags or JSON config, this can then easily be multi-threaded
...
+
+We are fiercely independent and not going anywhere. [Read more about our philosophy](https://www.runly.io/about/).
+
+Our mission is to create great developer experiences making it trivial to build and scale jobs while providing a great user experience in your app. We have plenty of ideas on how to do this but we need your support to continue working on this project.
```
Comment (WillSoss) [inline README.md:109]:
```suggestion
Our mission is to create great developer experiences by making it trivial to build and scale jobs in order to provide a great user experience in your app. We have plenty of ideas on how to do this but we need your support to continue working on this project.
```

**Comment by WillSoss** [comment]
@chadly The review tool wouldn't let me leave a comment on L39 since you didn't modify it, but I think we should change it to:

Here is a job that reads a CSV file (using [CsvHelper](https://joshclose.github.io/CsvHelper/)) and sends an email to each person in the file.

File: README.md (hunk 2 above)
Comment (chadly) [reply README.md:5]: I think we should leave "opinionated framework" in there. It conveys that we've though of the issues you are going to have before you have and have designed the API in a way that will give you the least friction ([stolen from here](https://stackoverflow.com/questions/802050/what-is-opinionated-software#:~:text=An%20opinionated%20framework%20is%20one,made%20by%20the%20framework%20designer.%20%E2%80%93)). Maybe we could spell that out more explicitly if you want?

Comment (WillSoss) [reply README.md]: I know this is a thing, but really everyone has an opinion that shows in their code. I feel like we don't need to hit people over the head with this, it should be obvious, and I wouldn't want to turn people off right away who look at opinionated software as bad software.

**Pull request merged**